packages/icons/icons.json

```
@@ -1969,182 +1969,182 @@
     {
         "displayName": "Annotation",
         "iconName": "annotation",
-        "tags": "note, comment, edit,",
+        "tags": "note, comment, edit, , edit, write",
         "group": "editor",
         "codepoint": 61717
     },
     {
         "displayName": "Pivot",
         "iconName": "pivot",
-        "tags": "rotate, axis",
+        "tags": "rotate, axis, arrow bounce down right, add node, change direction, redirect, node, branch, graph",
         "group": "action",
         "codepoint": 62071
     },
     {
         "displayName": "Ring",
         "iconName": "ring",
-        "tags": "empty, circle, selection",
+        "tags": "empty, circle, selection, hollow circle, empty circle, hole, disc",
         "group": "miscellaneous",
         "codepoint": 62102
     },
     {
         "displayName": "Heat grid",
         "iconName": "heat-grid",
-        "tags": "chart",
+        "tags": "chart, grid, heat map, squares, black and white",
         "group": "data",
         "codepoint": 61929
     },
     {
         "displayName": "Gantt chart",
         "iconName": "gantt-chart",
-        "tags": "bar chart, schedule, project",
+        "tags": "bar chart, schedule, project, step chart, stairs, waterfall",
         "group": "data",
         "codepoint": 61894
     },
     {
         "displayName": "Variable",
         "iconName": "variable",
-        "tags": "math, calculation",
+        "tags": "math, calculation, function input, variable input, parentheses, function, parameter, input",
         "group": "table",
         "codepoint": 62219
     },
     {
         "displayName": "Manual",
         "iconName": "manual",
-        "tags": "guide, instruction",
+        "tags": "guide, instruction, documentation, docs, book, open",
         "group": "interface",
         "codepoint": 62005
     },
     {
         "displayName": "Add row: top",
         "iconName": "add-row-top",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, add, add row",
         "group": "table",
         "codepoint": 61702
     },
     {
         "displayName": "Add row: bottom",
         "iconName": "add-row-bottom",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, widget, add",
```

```suggestion
        "tags": "table, attach, join, widget",
```

if this is a "widget", shouldn't "add-row-top" also be a widget? same for add-*-column icons. let's be consistent.